### DISCUSSION TITLE
Is it possible to return a list of files one page at a time?

### DISCUSSION THREAD

**Discussion by glen-84** [Studio-42/elFinder#3312] [Ideas]
For example, if the folder contains a large number of files, then it doesn't make sense to return all of these files in a single request. Is it possible for the client to request a certain number of files, with a specific offset or file to start from?

**Comment by nao-pon** [comment]
@glen-84 Since elFinder's current API performs sorting and incremental search on the client side, it is necessary to acquire item data at once. Since it is necessary to extend the API, it is a future issue.

**Comment by mitulpanch** [comment]
this will be usefull for gcp connector, gcp cloud storage api does not support more no of entries

**Comment by PawelPatyk** [comment]
It’s been nine years, and this important feature is still missing :(
If there are a lot of files in a folder, ElFinder simply returns an error after a while, instead of displaying a specific number of files with the option to view more.
